README.md

```diff
@@ -18,15 +18,12 @@ http://tsetse.info/api/
 First, install a current Bazel distribution.
 
 Add the `@bazel/typescript` npm package to your `package.json` `devDependencies`.
-Optionally add the `@bazel/karma` npm package if you would like to use the
```

Comment:
Since this is the main README, we should probably list the other rules sets contained and link to their READMEs:
`npm_bazel_karma`, `@bazel/karma` => `internal/karma/README.md`

Reply:
added a note at the very top about Karma having moved